images/images.go

```diff
@@ -0,0 +1,190 @@
+package image
+
+import (
+	"bytes"
+	"github.com/kolesa-team/go-webp/encoder"
+	"github.com/kolesa-team/go-webp/webp"
+	"github.com/nfnt/resize"
+	"image/jpeg"
+	"image/png"
+	"io"
+	"log"
+	"os"
+	"path"
+	"path/filepath"
+)
+
+var (
+	Webp = "webp"
+	Jpeg = "jpeg"
+	Png  = "png"
+)
```

Comment:
Better to introduce a new type like `ImageExtension` and use it. In this case only your pre-defined values can be used for this type e.g. no one can pass something like `"mp3"`

Here is a code example:
```go
type Status string

const (
  TransactionOpen  Status = "TRANSACTION_OPEN"
  TransactionFailed Status = "TRANSACTION_FAILED"
)
```